### ISSUE TITLE
add gh action that does line by line ss_summary.sso comparison

### ISSUE THREAD

**Issue by k-doering-NOAA** [nmfs-stock-synthesis/stock-synthesis#458]
Thanks @Rick-Methot-NOAA for this idea; this may help catch differences among models in quantities that are  not tested

The difficult part is that there will always be some differences (e.g., the date of the model run may be different , the number of digits or use of sci notation or not may be different), so it may be difficult to make this job completely automated.

A first step toward figuring out how this could be useful, though, is simply to develop a job that does the comparison.

**Comment by Rick-Methot-NOAA** [comment]
I do not think we need to run this with every commit.  Is is possible to create a gha dashboard so we could manually run when needed?

**Comment by kellijohnson-NOAA** [comment]
Yes, you can run it manually as long as we add `workflow_dispatch:` to `on:`.

**Comment by e-perl-NOAA** [comment]
Limited number of quantities are actually checked, however this would be difficult to systematically & automatically check all number. Could use r4ss function that reads the summary file to check if ratios are off by 5% then they can be flagged.
